study.md

```diff
@@ -43,21 +42,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+Developers are referring to the model-view-controller when referring to the word "view".  Though I'm still not entirely sure what the MVC is, it does work on a single application page in order to interact with a server to product api - content.  
```

Comment:
A view is what is rendered with specific data. Its what the user is seeing. A view-state is the state of that view, like what the data that is showing is, as opposed to different data for a different view